src/shapes/Object/Object.ts

```diff
@@ -70,6 +71,34 @@ export type TCachedFabricObject<T extends FabricObject = FabricObject> = T &
     _cacheContext: CanvasRenderingContext2D;
   };
 
+export type ObjectToCanvasElementOptions = {
+  format?: ImageFormat;
+  // Multiplier to scale by
+  multiplier?: number;
+  // Cropping left offset. Introduced in v1.2.14
+  left?: number;
+  // Cropping top offset. Introduced in v1.2.14
+  top?: number;
+  // Cropping width. Introduced in v1.2.14
+  width?: number;
+  // Cropping height. Introduced in v1.2.14
+  height?: number;
+  // Enable retina scaling for clone image. Introduce in 1.6.4
+  enableRetinaScaling?: boolean;
+  // Remove current object transform ( no scale , no angle, no flip, no skew ). Introduced in 2.3.4
+  withoutTransform?: boolean;
+  // Remove current object shadow. Introduced in 2.4.2
+  withoutShadow?: boolean;
+  // Account for canvas viewport transform
+  viewportTransform?: boolean;
+  // Function to create the output canvas to export ont
+  canvasProvider?: (el?: HTMLCanvasElement) => Canvas;
+}
```

Comment:
I think if you use `/** */` as comment syntax then it appears when you hover the option in VSCODE